CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # `chainweb-node` Changelog
 
+## 1.4 (2019-12-14)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2019-12-17T00:00:00Z**. Note administrators must upgrade to this version before
+that date.
+
+This version will stop working on **2020-01-15T00:00:00Z**. Node administrators must
```

Comment:
This para is dupe of above

Reply:
It's different. The former is about the upgrade to 1.4 from previous versions. The later is about the upgrade from this version to the next version.

But I am not happy with these paragraphs neither. @mightybyte may you, as a native, speaker can make this better?